recovery-contracts/TransferHelper.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// source: https://github.com/Uniswap/solidity-lib/blob/master/contracts/libraries/TransferHelper.sol
+pragma solidity 0.8.10;
+
+// helper methods for interacting with ERC20 tokens and sending ETH that do not consistently return true/false
+library TransferHelper {
+  function safeTransfer(
+    address token,
+    address to,
+    uint256 value
+  ) internal {
+    // bytes4(keccak256(bytes('transfer(address,uint256)')));
+    (bool success, bytes memory data) = token.call(
+      abi.encodeWithSelector(0xa9059cbb, to, value)
+    );
+    require(
+      success && (data.length == 0 || abi.decode(data, (bool))),
+      'TransferHelper::safeTransfer: transfer failed'
+    );
+  }
+}
```

Comment:
don't we have any tests for these recovery contracts?

Reply:
the solidity versions for the old and new recovery contracts are different therefore I'm are not able to compile the new versions for writing tests